superset-frontend/cypress-base/cypress/integration/dashboard/load.test.js

```diff
@@ -43,14 +43,19 @@ describe('Dashboard load', () => {
     slices.forEach(slice => {
       const vizType = slice.form_data.viz_type;
       const isLegacy = isLegacyChart(vizType);
-      // TODO(villebro): enable V1 charts
       if (isLegacy) {
         const alias = `getJson_${slice.slice_id}`;
         const formData = `{"slice_id":${slice.slice_id}}`;
         const route = `/superset/explore_json/?*${formData}*`;
         cy.route('POST', `${route}`).as(alias);
         aliases.push(`@${alias}`);
       }
+      else {
+        const alias = `getV1ChartJson`;
+        const route = `/api/v1/chart/data?*`;
+        cy.route('POST', `${route}`).as(alias);
+        aliases.push(`@${alias}`);
+      }
```

Comment:
As the legacy/V1 routes are always the same, I wonder if it would make sense to add a function to get the route to the `vizPlugins.ts` utility?

Reply:
If it is always the same and i put * at the end of the route -> I see benefit in suggested solution :)